src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -201,7 +201,31 @@ void GNAPlugin::copyInputData(T *dst,
     }
 }
 
-void GNAPlugin::ExportScores(void *ptr_dst,
+template <typename T>
+void GNAPlugin::UpscaleAndCast(T* dst_ptr, const uint8_t* input_ptr, const Precision& precision_in, const float& scale_factor) {
```

Comment:
IMO U**n**scale was a better name ;)
Upscale suggest that the resulting number is greater,
but input is **divided** by scale_factor
typically scale_factor is greater than 1

`CastAndDivide` is also reasonable name